### DISCUSSION TITLE
Emily CSS Text Portrait

### DISCUSSION THREAD

**Discussion by warengonzaga** [warengonzaga/css-text-portrait-builder#18] [Showcase]
I created this builder before I create my own version of the CSS text portrait. 😎
The first reason is to enhance my skills and the second is to help others to create their own without digging too much in CSS.
Also, eliminate bad practices in writing CSS lol. 😅 (I'm saying I'm perfect lol).

Here's a little showcase for my girlfriend and future wife. 😎

![msedge_5XHX4P60eg](https://user-images.githubusercontent.com/15052701/142731659-2911736d-6c84-485e-9dc0-e7e95d746921.png)

I made a few customizations that are outside of the builder.
I will add these changes to the coming versions of the project. 🚀

**Comment by simp4Ingrid** [comment]
poide rin po magpagawa? im too much struggiling po from the steps im not a pro po ee

**Reply by warengonzaga** [reply]
A new video tutorial in Tagalog will be available later. Please follow my update here: <https://facebook.com/warengonzagaofficial>
